### PR TITLE
Final styling

### PR DESCRIPTION
**Pull request by MaggieSchuetz**

### CODE REVIEW

File: src/components/Form.js [hunk 1]
```diff
@@ -181,85 +182,92 @@ function Form({
...
+          <Input
+            id="photoUpload"
+            name="photoUpload"
+            className="photoUpload"
+            type="file"
+            multiple
+            onChange={e => {
+              setImages([...e.target.files]);
+            }}
+          />{' '}
```
Comment (ttaannjjaa) [inline src/components/Form.js:227]: probably you can remove this

File: src/components/Form.js [hunk 2]
```diff
@@ -181,85 +182,92 @@ function Form({
...
+            name="title"
+            type="text"
+            placeholder="Title"
+            value={title}
+            maxLength="80"
+            required
+            onChange={handleTitleChange}
+          />
+        </Container>
+        {messageTitle && <P className="message">{messageTitle}</P>}
```
Comment (ttaannjjaa) [inline src/components/Form.js:215]: Do you need the className - Attribut in addition to styled components? Also in line 216 and 268?

**Review by ttaannjjaa — APPROVED** [review]
great work, looks all very good to me, just 2 little comments

**Review by teifler — COMMENTED** [review]
Looks good, really nice code :)

**Pull request merged**